src/webextension/widgets/text-area.js

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I recommend there be a .css for textrea that specifies `resize: none;`.



I really don't think we'll have a valid scenario where a resizable textarea (without resizing its container) is desired.